src/System.Text.Primitives/System/Text/Formatters/Utf16Formatter_time.cs

```diff
@@ -199,17 +199,16 @@ private static bool TryFormatO(DateTime value, TimeSpan offset, Span<byte> buffe
 
             if (kind == DateTimeKind.Local)
             {
-                int hours = offset.Hours;
                 char sign = Plus;
 
-                if (offset.Hours < 0)
+                if (offset < default(TimeSpan) /* a "const" version of TimeSpan.Zero */)
```

Comment:
is there an actual significant performance win for using default(TimeSpan) in place of TimeSpan.Zero?  If so, is it worth creating a codegen bug on this to see what improvements could be made here?